submissions/vovan-zt/popup/index.html

```diff
@@ -0,0 +1,71 @@
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>popup</title>
+</head>
+<body>
+    <div class="contaiter">
+        <nav class="popup" >
+            <ul class="popup__menu">
+                <li  class="popup__list">
+                    <input class="popup__check_input" type="checkbox" >
+                    <label><img class="popup__link" src="icons/bell.png" alt="img"></label>  
+                </li>
+
+                <li  class="popup__list">
+                    <input class="popup__check_input" type="checkbox" >
+                    <label><img class="popup__link" src="icons/bell.png" alt="img"></label> 
+                </li>
+                
+                <li  class="popup__list">
+                    <input  class="popup__check_input" type="checkbox" name="modal" id="modal">
+                    <label  for="modal" ><img class="popup__link" src="icons/popup-button.png" alt="img"></label> 
+
+                    <div class="modal">
+                        <ul class="modal__block">
+                            <li class="modal__list"><a href="#"> <img src="icons/apps.png" alt="img"><span>item1</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/blogs.png" alt="img"><span>item2</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/calendar.png" alt="img"><span>item3</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/chat.png" alt="img"><span>item4</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/classroom.png" alt="img"><span>item5</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/collection.png" alt="img"><span>item6</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/contacts.png" alt="img"><span>item7</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/documents.png" alt="img"><span>item8</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/files.png" alt="img"><span>item9</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/mail.png" alt="img"><span>item10</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/maps.png" alt="img"><span>item11</span></a></li>
+                            <li class="modal__list"><a href="#"> <img src="icons/mars.png" alt="img"><span>item12</span></a></li>
+                            
+                            <input  class="popup__check_input" type="checkbox" name="" id="modal2">
+                            <label  for="modal2" class="popup__link2"></label> 
```

Comment:
When an additional part of the popup is open, the button `Up` is not accessible from keyboard.

Reply:
Исправил, переключается на него после 12 элемента. По-другому не знаю как сделать.

Reply:
The current order of focused elements is not obvious to users but with JS it can be done easily, so I think that's ok, but you need to check your markup on the [validator](https://validator.w3.org/)